Arch-specific files go to /usr/lib rather than /usr/share.
Index: collab-maint.git/source/creator/CMakeLists.txt
===================================================================
--- collab-maint.git.orig/source/creator/CMakeLists.txt	2010-08-21 05:03:12.000000000 +0200
+++ collab-maint.git/source/creator/CMakeLists.txt	2010-08-21 05:04:26.000000000 +0200
@@ -71,7 +71,7 @@
 ENDIF(NOT WITH_SDL)
 
 IF(UNIX AND NOT APPLE)
-	SET(BLENDERPATH ${CMAKE_INSTALL_PREFIX}/share/blender/${BLENDER_VERSION})
+	SET(BLENDERPATH ${CMAKE_INSTALL_PREFIX}/lib/blender/${BLENDER_VERSION})
 	# blender_path in creator.c
 	ADD_DEFINITIONS(-DBLENDERPATH="${BLENDERPATH}")
 ENDIF(UNIX AND NOT APPLE)
